.github/workflows/pytest.yml

```diff
@@ -4,10 +4,12 @@ name: pytest
 on:
   pull_request:
     types:
-      - created
       - opened
       - synchronize
       - ready_for_review
```

Comment:
I bet we can actually stop triggering on these specific events, though since everything *but* `ci-integration` is free I think it's probably fine to keep it for now.

Reply:
I think we only included `ready_for_review` so that when we switched out of draft mode the integration tests would get run, but since we don't need to do that in the new setup, it seems like it would be nice to skip it, since then you can potentially merge immediately if all of the other requirements have been satisfied instead of needing to wait 5 minutes for the faster CI steps to complete.  It looks like [the default types](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) are `opened`, `synchronize`, and `reopened` which seem right to me so I think removing the `types` section here makes sense.